src/cli/chmod.js

```diff
@@ -49,24 +49,20 @@ module.exports = {
 
   handler (argv) {
     const {
+      ctx: { ipfs },
```

Comment:
Any reason why this is passed as `argv.ctx.ipfs` and not just `argv.ipfs`?

Reply:
ctx has a bunch of props like isDaemon, repoPath, print etc basically a container for all the extra context we inject into cmds to avoid collisions.